src/vs/workbench/contrib/terminalContrib/stickyScroll/browser/terminalStickyScrollOverlay.ts

```diff
@@ -191,10 +192,17 @@ export class TerminalStickyScrollOverlay extends Disposable {
 			this._pendingShowOperation = true;
 			this._show();
 		} else {
+			this._cancelPendingShow();
 			this._hide();
 		}
 	}
 
+	private _cancelPendingShow(): void {
+		const showFn = this._show as typeof this._show & { clear: () => void };
```

Comment:
I mean to cancel debounced `_show` when hide is called.
Maybe I can just put this function as part of `hide()` entirely.